### ISSUE TITLE
FNX3-17135 ⁃ [Bug][a11y] Three-dot menu is reopened when double tapped anywhere outside the opened menu

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#3584]
## Steps to reproduce
1. Turn Talkback on.
1. Tap on three-dot menu.
1. Exit the menu double-tapping outside it.

### Expected behavior
Double tapping outside the menu should dismiss the menu.

### Actual behavior
The three-dot menu is opened again and again when tapping anywhere outside the menu.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8)
* Fenix version: 1.0.0-rc.2 (#11700147)

![menu1](https://user-images.githubusercontent.com/40694158/59847590-9a3d4080-936b-11e9-8e4e-3184caf47861.gif)

**Comment by pdjstone** [comment]
I get a similar issue on my phone - if I double-tap the three-dot menu button, it opens twice (you have to complete the double-tap before the menu opens, a slower device probably helps). Two separate taps outside the menu are needed to close both.

**Comment by mihai-adrian** [comment]
I think the first issue mentioned is actually the correct way to respond to a double tap, when a button is selected. The official documentation mentions:"When the focus reaches an item you'd like to select, double-tap **anywhere** on the screen to select the focused item" (https://support.google.com/accessibility/android/answer/6006598?hl=en).

The second issue mentioned is that due to the above functionality, after a double tap, the button still has focus and has the same onClickListener, and if you double tap it another settings pop-up is shown.

There are three approaches to fix this : 
- change the listener to dismiss the settings pop-up when it is already opened (chrome, skype and others do that)
- check if popup is already shown and prevent showing a second one
- remove focus from button after pop-up is shown.

Which one would be the preferred approach?

**Comment by lime124** [comment]
I think this is more of an eng ask of how they want to fix the bug? /cc @boek

**Comment by DiegoARL38** [comment]
As a regular screen reader user, dismissing the menu if already open is what I expect, since most applications do this.

**Comment by sv-ohorvath** [comment]
Verified as fixed. The settings option is selected by default now when you open the 3 dot menu.

**Comment by Mugurell** [comment]
Closing as per https://github.com/mozilla-mobile/fenix/issues/3584#issuecomment-541715484
